src/storage/DomainDecomposition.py

```diff
@@ -54,9 +54,9 @@
 
```

Comment:
Please update the documentation section with the information about the new parameter.